build.xml

```diff
@@ -1027,6 +1028,22 @@
     </copy>
   </target>
 
+  <macrodef name="checktestnameshelper">
+    <sequential>
+      <taskdef name="test-name-check_" classname="org.apache.cassandra.anttasks.TestNameCheckTask" classpath="${test.classes}">
+        <classpath>
+          <path refid="cassandra.classpath.test"/>
+          <path location="${fqltool.build.classes}"/>
+        </classpath>
+      </taskdef>
+      <test-name-check_/>
+    </sequential>
+  </macrodef>
```

Comment:
nit: what about adding these changes into `.build/build-linter.xml` file (along with all the existing checkstyle and linter stuff)?  including standardising the way we skip the linter tasks.

this was discussed a little in CASSANDRA-18077

Reply:
Interesting, I will check.

Reply:
@michaelsembwever I would really keep this as is. Please keep in mind this goes to 5 branches already and each branch is little bit different etc. I am spending a lot of time on merging this up from 3.0. I think what you suggested might be explored but I would really reserve this work for another ticket with all other checkstyle targets.